secp256kfun/README.md

```diff
@@ -35,7 +35,7 @@ secp256kfun = "0.2"
 
 ### Should use?
 
-This library is ready for production as long what you are trying to produce is **fun and amusement!**.
+This library is ready for production as long what you are trying to produce is for **fun and amusement!**.
```

Comment:
```suggestion
This library is ready for production as long what you are trying to produce is **fun and amusement!**.
```
"is fun and amusement" is correct here.

Reply:
Is this a meme I missed? I won't be offended if it is, I miss a lot of memes and internet jokes :)

Reply:
oh now I read it, my bad.